ksqldb-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java

```diff
@@ -171,8 +177,10 @@ private void recordMessagesConsumed(final double value) {
     this.messagesIn.record(value);
   }
 
-  private void recordTotalBytesConsumed(final double value) {
+  private void recordTotalBytesConsumedAndPerSecond(final double value) {
```

Comment:
nit: the title feels kinda clunky, I know what you mean but it took me a second to understand. Maybe something like `recordTotalAndPerSecondBtyesConsumed`?